### DISCUSSION TITLE
Does my Hydra command look right?

### DISCUSSION THREAD

**Discussion by baseball347** [facebookresearch/hydra#2766] [Q&A]
I have a Facebook account where I do not know the email and do not have access to the email either. I also do not know the password. I am going to see if I can try to crack the password with Hydra. I am not sure if I have the right syntax for the command I need to run. Can someone please take a look at it? I would appreciate it!

hydra -l username -P /usr/share/wordlists/rockyou.txt.gz [www.facebook.com](http://www.facebook.com/) http-post-form "/www.facebook.com:username=USER&password=^PASS^:The password you’ve entered is incorrect."

I am not using the word USER within the syntax. I am actually using my Facebook username.

**Comment by Jasha10** [comment]
@baseball347 you've commented on the wrong repository.
This repository is for the Hydra configuration management software (https://hydra.cc).
You're looking for the hydra password cracker (https://github.com/vanhauser-thc/thc-hydra)

**Comment by baseball347** [comment]
Thanks!
